src/components/schedule-page/table/schedule/schedule.component.tsx

```diff
@@ -47,65 +52,69 @@ export function ScheduleComponent({
-                    </tbody>
-                  </table>
-                </td>
-                <td className="summaryContainer">
-                  <SummaryTableComponent
-                    uuid={scheduleLocalState.uuid}
-                    data={scheduleLocalState.nurseShiftsSection}
-                    workerType={WorkerType.NURSE}
-                  />
-                </td>
-              </tr>
+              {areNursesPresent() && (
+                <tr className="sectionContainer">
+                  <td>
+                    <NameTableComponent
+                      uuid={scheduleLocalState.uuid}
+                      workerType={WorkerType.NURSE}
+                      data={scheduleLocalState.nurseShiftsSection}
+                      clickable={true}
+                    />
+                  </td>
+                  <td>
+                    <table>
+                      <tbody className="table" data-cy="nurseShiftsTable">
+                        <ShiftsSectionComponent
+                          uuid={scheduleLocalState.uuid}
+                          workerType={WorkerType.NURSE}
+                          data={scheduleLocalState.nurseShiftsSection}
+                        />
+                      </tbody>
+                    </table>
+                  </td>
+                  <td className="summaryContainer">
+                    <SummaryTableComponent
+                      uuid={scheduleLocalState.uuid}
+                      data={scheduleLocalState.nurseShiftsSection}
+                      workerType={WorkerType.NURSE}
+                    />
+                  </td>
+                </tr>
+              )}
             </ScheduleFoldingSection>
 
             <ScheduleFoldingSection name="Opiekunowie">
-              <tr className="sectionContainer">
-                <td>
-                  <NameTableComponent
-                    uuid={scheduleLocalState.uuid}
-                    workerType={WorkerType.OTHER}
-                    data={scheduleLocalState.babysitterShiftsSection}
-                    clickable={true}
-                  />
-                </td>
-                <td>
-                  <table>
-                    <tbody className="table" data-cy="otherShiftsTable">
-                      <ShiftsSectionComponent
-                        uuid={scheduleLocalState.uuid}
-                        workerType={WorkerType.OTHER}
-                        data={scheduleLocalState.babysitterShiftsSection}
-                      />
-                    </tbody>
-                  </table>
-                </td>
-                <td className="summaryContainer">
-                  <SummaryTableComponent
-                    uuid={scheduleLocalState.uuid}
-                    data={scheduleLocalState.babysitterShiftsSection}
-                    workerType={WorkerType.OTHER}
-                  />
-                </td>
-              </tr>
+              {areBabysittersPresent() && (
```

Comment:
Może lepiej od razu chować całą folding section?
Czyli przenieść ten and o jedną linijkę wyżej

Reply:
Mi się osobiście podoba bardziej z folding sections, ale nie jakoś silniej

Reply:
To niech będzie z sekcjami